secrets/drivertest/drivertest.go

```diff
@@ -54,6 +64,16 @@ func RunConformanceTests(t *testing.T, newHarness HarnessMaker) {
 	t.Run("TestDecryptMalformedError", func(t *testing.T) {
 		testDecryptMalformedError(t, newHarness)
 	})
+	t.Run("TestAs", func(t *testing.T) {
```

Comment:
You can also add a conformance `AsTest` that verifies `As` fails when passed `nil`:

https://github.com/google/go-cloud/blob/master/runtimevar/drivertest/drivertest.go#L72-L93
https://github.com/google/go-cloud/blob/master/runtimevar/drivertest/drivertest.go#L119-#L119

Reply:
Done.